crates/matrix-sdk-indexeddb/src/crypto_store/migrations.rs

```diff
@@ -52,7 +108,11 @@ pub async fn open_and_upgrade_db(name: &str) -> Result<IdbDatabase, DomException
             create_stores_for_v5(evt.db())?;
         }
 
-        info!(old_version, new_version, "IndexeddbCryptoStore upgrade complete");
+        if old_version < 6 {
+            create_stores_for_v6(evt.db())?;
+        }
+
+        info!(old_version, new_version, "IndexeddbCryptoStore upgrade phase 1 complete");
```

Comment:
Can we add a big, explicit comment here telling new schema upgrades shouldn't go to this function, but to the most recent one, instead? (I for one would forget otherwise, and put new things in there)